test/test_libs.py

```diff
@@ -1769,6 +1781,37 @@ def test_collector_heterogeneous(self, n_envs=10, frames_per_batch=20):
         assert env.reward_key not in _td.keys(True, True)
         assert env.action_key not in _td["next"].keys(True, True)
 
+    @pytest.mark.parametrize("n_agents", [1, 5])
+    def test_grouping(self, n_agents, scenario_name="dispersion", n_envs=2):
+        env = VmasEnv(
+            scenario=scenario_name,
+            num_envs=n_envs,
+            n_agents=n_agents,
+        )
+        env = VmasEnv(
+            scenario=scenario_name,
+            num_envs=n_envs,
+            n_agents=n_agents,
+            # Put each agent in a group with its name
+            group_map={
+                agent_name: [agent_name] for agent_name in reversed(env.agent_names)
+            },
+        )
+
+        # CHeck that when setting the action for a specific group that is reflected to the right agent in the backend
+        for group in env.group_map.keys():
+            env.reset()
+            action = env.full_action_spec.zero()
+            action[group, "action"] += 1.0
```

Comment:
```suggestion
            action.get((group, "action")) += 1.0
```
although we should avoid inplace modification in tensordicts like this (eg, if you're working with a lazy stack it won't do anything)